src/coreclr/vm/comdelegate.cpp

```diff
@@ -189,16 +189,57 @@ class ShuffleIterator
         if (m_currentByteStackIndex < m_argLocDesc->m_byteStackSize)
         {
             const unsigned byteIndex = m_argLocDesc->m_byteStackIndex + m_currentByteStackIndex;
```

Comment:
Include this in the `#if` since it is not used anywhere else in this branch.

Reply:
It is used in the else branch.